HarmonyCore/FileIO/IDataObjectIO.dbl

```diff
@@ -38,6 +38,11 @@ namespace Harmony.Core.FileIO
 		;;; </summary>
 		readonly property NumberOfKeys, int
 
+		;;; <summary>
+		;;;  Specify the maximum amount of seconds to wait for lock operations to complete.
+		;;; </summary>
+		readwrite property TimeoutSeconds, double, 5
```

Comment:
it looks like this is an interface, I dont think you can specify a default value for a property of an interface. This might need to be reported as a compiler bug.

Reply:
Resolved. See 367609d81121cff4ed842e9e0ff4fcc0e48eff74